third_party/README.md

```diff
@@ -48,6 +48,14 @@ Usually the process is
 
 Updating some dependencies requires extra care.
 
+### Updating third_party/abseil-cpp
+
+- Two additional steps should be done before running `generate_projects.sh` above.
```

Comment:
Sg. Instructions are useful. But it would be nice to be able to get rid of these extra steps - how hard would it be now that our Makefile has ben shrinked significantly and the extract_metadata_from_bazel.py script has stabilized?

Reply:
Whoa, just looked at https://github.com/grpc/grpc/blob/master/src/abseil-cpp/preprocessed_builds.yaml.gen.py - VERY complicated (and looks like it does similar sort of magic that's already done by extract_metadata_from_bazel.py)
Maybe this would be a nice opportunity for simplification.

Reply:
Yeah, it's a bit complicated and that's why I made the separate script to keep our build scripts from getting way more complicated. Since it's dealing with the external build files, I like to isolate it from ours.